obspy/core/inventory/response.py

```diff
@@ -735,10 +736,227 @@ def __init__(self, resource_id=None, instrument_sensitivity=None,
             msg = "response_stages must be an iterable."
             raise ValueError(msg)
 
-    def get_evalresp_response_for_frequencies(
-            self, frequencies, output="VEL", start_stage=None, end_stage=None):
+    @classmethod
+    def from_resp(cls, resp, frequency=None, sr=None):
```

Comment:
Could you please move this to `io.xseed.parser.Parser.get_response()`? We should try and avoid "polluting" our core objects with specific data sources if possible.